docs/howto/build-guides/platforms.rst

```diff
@@ -0,0 +1,169 @@
+Build platform-specific charms
+==============================
+
+A ``charmcraft.yaml`` defines where charms are built and where charms are run
+using the ``platforms``, ``base``, and ``build-base`` keywords.
+
+See the :doc:`Platforms reference </reference/platforms>` for more information.
+
```

Comment:
To be pedantic, the first section should probably be a simple task about setting the `base`, and ignore `platforms` entirely.